projects/docs/src/app/common/dynamic-form/model/dynamicFormType.ts

```diff
@@ -0,0 +1,11 @@
+export type DynamicFormType = "Text" | "TextArea" | "Checkbox" | "Dropdown" | "Datepicker" | "Radio" | "Number";
+
+export const DynamicFormType = {
+    Text: "Text" as DynamicFormType,
+    TextArea: "TextArea" as DynamicFormType,
+    Checkbox: "Checkbox" as DynamicFormType,
+    Dropdown: "Dropdown" as DynamicFormType,
+    Datepicker: "Datepicker" as DynamicFormType,
+    Radio: "Radio" as DynamicFormType,
+    Number: "Number" as DynamicFormType,
```

Comment:
Can't you just use Typescript's `Pick` type??
```ts
Text: Pick<DynamicFormType, "Text">;
```

Reply:
this is how openapi generates automatically so I just left it like that since it works

Reply:
Pick will not work as pick will pick from an interface type